### PR TITLE
Updating presto to version 323

### PR DESCRIPTION
**Pull request by ChethanUK**
- Software Requirements for Presto 323 is Java JRE 11 (64-bit) - OpenJDK

### CODE REVIEW

File: starburst-presto/presto.sh [hunk 1]
```diff
@@ -50,9 +70,9 @@ function wait_for_presto_cluster_ready() {
 # Download and unpack Presto Server
 function get_presto() {
   wget -nv --timeout=30 --tries=5 --retry-connrefused -O - \
-    "${PRESTO_BASE_URL}/${PRESTO_MAJOR_VERSION}e/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz" |
+    "${PRESTO_BASE_URL}/${PRESTO_MAJOR_VERSION}e/${STARBURST_PRESTO_VERSION}/presto-server-${STARBURST_PRESTO_VERSION}.tar.gz" |
```
Comment (medb) [inline starburst-presto/presto.sh]: Where `STARBURST_PRESTO_VERSION` variable is defined?

**Comment by medb** [comment]
This PR has too many unrelated changes.

**Comment by ChethanUK** [comment]
> too many unrelated changes.

ya it's because I just cherry-pick one commit from the local repo

**Pull request closed without merging**